log/term/terminal_windows.go

```diff
@@ -8,15 +8,23 @@
 package term
 
 import (
+	"bytes"
+	"encoding/binary"
 	"io"
+	"regexp"
 	"syscall"
 	"unsafe"
+
+	"golang.org/x/sys/windows"
```

Comment:
No, it requires using `go get` to retrieve and would need to be vendored by those that vendor their dependencies.